### ISSUE TITLE
unnecessary_getters_setters should catch wrapping a private final field

### ISSUE THREAD

**Issue by natebosch** [dart-lang/linter#1991]
Bad:

```dart
class Foo {
  final _foo = 'foo';
  String get foo => _foo; // unnecessary_getters_setters
}
```

Good:

```dart
class Foo {
  final foo = 'foo';
}
```

**Comment by bwilkerson** [comment]
I'm not sure why, but we appear to have a separate lint for that (unnecessary_getters).

**Comment by pq** [comment]
It looks like `unnecessary_getters` is disabled (see: #23).  Not sure if we want to resuscitate it but I'm happy to discuss.
